.circleci/config.yml

```diff
@@ -330,6 +330,22 @@ jobs:
                 name: Tag
                 command: |
                   ./ops/scripts/ci-docker-tag-op-stack-release.sh <<parameters.registry>>/<<parameters.repo>> $CIRCLE_TAG $CIRCLE_SHA1
+      - when:
+          condition: "<<parameters.release>>"
+          steps:
+            - gcp-oidc-authenticate:
+                service_account_email: GCP_SERVICE_ATTESTOR_ACCOUNT_EMAIL
+            - run:
+                name: Sign
+                command: |
+                  SIGNER_FOLDER="./.circleci/signer"
+                  export IMAGE_PATH="<<parameters.registry>>/<<parameters.repo>>/<<parameters.docker_name>>:<<parameters.docker_tags>>"
```

Comment:
It would be more ideal to include as much of this functionality in a bash script as possible and then just call that script with args/env vars set in the context

Reply:
Added functionality into the script to retrieve the digest.